test/src/e2e.long/mempool.test.ts

```diff
@@ -80,6 +81,75 @@ describe("Memory pool size test", function() {
             expect(pendingTransactions.length).to.equal(sizeLimit);
         }).timeout(20_000);
 
+        it("Rejected by limit and reaccepted", async function() {
+            const sent = [];
+            for (let i = 0; i < sizeLimit * 2; i++) {
+                sent.push(
+                    await nodeA.sendPayTx({
+                        seq: i,
+                        awaitInvoice: false
+                    })
+                );
+            }
+            let counter = 0;
+            while (
+                (await nodeB.sdk.rpc.chain.getPendingTransactions()).length <
+                sizeLimit
+            ) {
+                await wait(500);
+                counter += 1;
+            }
+
+            const pendingTransactions = await nodeB.sdk.rpc.chain.getPendingTransactions();
+            const pendingTransactionHashes = pendingTransactions.map(
+                (tx: SignedTransaction) => tx.hash().value
+            );
+            const rejectedTransactions = sent.filter(
+                tx => !pendingTransactionHashes.includes(tx.hash().value)
+            );
+
+            await nodeB.sdk.rpc.devel.startSealing();
+
+            counter = 0;
+            while (
+                (await nodeB.sdk.rpc.chain.getPendingTransactions()).length > 0
+            ) {
+                await wait(500);
+                counter += 1;
+            }
+
+            await nodeB.sdk.rpc.devel.stopSealing();
+
+            Promise.all(
```

Comment:
Sorry. I missed using `await`. It should be `await Promise.all(` if you want to wait to send.
But it seems that you don't need to wait for it. Because you count the number of pending transactions on line 131.

Reply:
Then I think It's better to add await in front of Promise.all and remove the waiting until the number of pending transactions and rejected transactions become equal.
The last part of test check all rejected transactions are now in pending transactions, it automatically check the number of two arrays are same.

Reply:
Would you check when RPC responses for chain_sendSignedTransaction?